roles/cobbler/templates/triggers/install/post/cephlab_ansible.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Yeah, /bin/bash works on RHEL 7 and Ubuntu 14.04; if it starts failing on older stuff I'll figure something out.